### PR TITLE
Add ability to specify endpoint

### PR DESCRIPTION
**Pull request by thomasyu888**
Add ability to specify the data endpoint as can't always assume that the endpoint will be hosted through local host.

### CODE REVIEW

File: nlpsandboxclient/cli/community.py [hunk 1]
```diff
@@ -22,9 +22,12 @@ def get_num_users():
 
 @cli.command()
 @click.option('--output', help='Output json filepath', type=click.Path())
-def get_clinical_notes(output):
+@click.option('--data_endpoint', help='Data endpoint')
```
Comment (tschaffter) [inline nlpsandboxclient/cli/community.py:25]: - Do not use "data_" as a synonym of "data_node_"

The term `endpoint` is usually used to refer to the full URL of a request, i.e. nothing expected after the _end_point. Here I think that you are using this information as the base of the endpoint.

The OpenAPI provides some hints regarding how we should name this term.

image

[Source](https://swagger.io/docs/specification/2-0/api-host-and-base-path/)

Let's have a live chat to settle on a naming convention given that we will use them terms countless times in our projects. I would like also to identify how best handle this information when this package is used with the CLI and as a python dependency. I'll setup a brief call.

**Review by tschaffter — CHANGES_REQUESTED** [review]

File: nlpsandboxclient/cli/community.py (hunk 1 above)
Comment (thomasyu888) [reply nlpsandboxclient/cli/community.py:25]: Ya, I have always struggled with the terminology.  One thing we can technically do is break it down into three, what we are looking for is the `scheme`, `host` and `basePath`.

Comment (thomasyu888) [reply nlpsandboxclient/cli/community.py]: Decided to use `data_node_host`

**Review by tschaffter — APPROVED** [review]
👍

**Pull request merged**